db/migrate/20230118132147_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :author, foreign_key: { to_table: :users}
+  end
```

Comment:
- [ ] You have done a great job to add references of likes to a user. Kindly also note that you are also required to add a reference to the post because a like will belong to both a user and a post. KIndly include it.

**From the provided ERD** you note that you have both userID and postID

![image](https://user-images.githubusercontent.com/86318284/213198347-91e02499-a62a-497f-a8f6-2be0f8a91170.png)

Reply:
![image](https://user-images.githubusercontent.com/105387310/213215967-55071709-e9e2-4982-b68f-a2d3acb18907.png)

Ok, I will fix it!